app/proguard-rules.pro

```diff
@@ -119,4 +119,5 @@
 
 # Keep Android Lifecycle methods
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1596302
--keep class androidx.lifecycle.** { *; }
\ No newline at end of file
+-keep class androidx.lifecycle.** { *; }
+-keep class okhttp3.** { *; }
```

Comment:
nit: since we didn't need to add okhttp as a dependency, I assume we will not need the keep rule either?